presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java

```diff
@@ -46,9 +46,10 @@ protected void setup(Binder binder)
                     .addBinding()
                     .to(LdapFilter.class)
                     .in(Scopes.SINGLETON);
+            checkState(ldapServerConfig.getServerType() != null, "Missing property 'authentication.ldap.server-type'");
+            checkState(ldapServerConfig.getUserObjectClass() != null, "Missing property 'authentication.ldap.user-object-class'");
```

Comment:
hmm..the property user-object-class is mandatory only for group membership check, and not for password validation, right?. so maybe we shud check if group-dn is set first